tests/mocha/index.html

```diff
@@ -32,6 +32,20 @@
       <block type="basic_block"></block>
     </xml>
 
+    <xml id="toolbox-simple" style="display: none">
```

Comment:
I don't think this is being used anywhere? Is the plan for this to be used for a future use case?

Reply:
Yes, there was a plan, but I'll remove it now and add it back later if I want it again.